redisson/src/main/java/org/redisson/executor/CronExpression.java

```diff
@@ -32,24 +32,21 @@
 
 package org.redisson.executor;
 
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 import java.io.Serializable;
 import java.text.ParseException;
-import java.util.Calendar;
-import java.util.Date;
-import java.util.HashMap;
-import java.util.Iterator;
-import java.util.Locale;
-import java.util.Map;
-import java.util.SortedSet;
-import java.util.StringTokenizer;
-import java.util.TimeZone;
-import java.util.TreeSet;
+import java.util.*;
+
+import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.ANY;
 
 /**
  * @author Sharada Jambula, James House
  * @author Contributions from Mads Henderson
  * @author Refactoring from CronTrigger to CronExpression by Aaron Craven
  */
+@JsonAutoDetect(creatorVisibility = ANY, fieldVisibility = ANY)
```

Comment:
@arpit728 Did you check it with different codecs not only Jackson?

Reply:
As of now I have only tested it with Jackson. Do we have unit tests for this or I will need to manually test it for all the codecs?

Reply:
@arpit728 

Please rollback all changes for this class.